packages/fxa-settings/src/components/FormPassword/index.stories.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Subject } from './mocks';
+import { LocationProvider } from '@reach/router';
+import AppLayout from '../AppLayout';
+import FormPassword from '.';
+
+export default {
+  title: 'components/FormPassword',
+  component: FormPassword,
+};
```

Comment:
Just noticed I need to add `as Meta` here and in the other stories to make eslint happy.